public/img/GearIcon.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="228" height="228" viewBox="0 0 228 228" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
hmm just a nit but I think it might be better to have the icons as png files since it's not really a flat or simple vector shape that needs to be scaled (good for svg) and the level of detail might suit png better ([according to this article lol](https://www.katescott.co/blog/jpg-vs-png-vs-gif-vs-svg-image-file-formats))